packages/ember-glimmer/lib/ember-views/component.js

```diff
@@ -0,0 +1,26 @@
+import CoreView from 'ember-views/views/core_view';
+import ChildViewsSupport from './child-views-support';
+import ViewStateSupport from 'ember-views/mixins/view_state_support';
+import ClassNamesSupport from 'ember-views/mixins/class_names_support';
+import InstrumentationSupport from 'ember-views/mixins/instrumentation_support';
+import AriaRoleSupport from 'ember-views/mixins/aria_role_support';
+import ViewMixin from 'ember-views/mixins/view_support';
+import EmberView from 'ember-views/views/view';
+
+export default CoreView.extend(
+  ChildViewsSupport,
+  ViewStateSupport,
+  ClassNamesSupport,
+  InstrumentationSupport,
+  AriaRoleSupport,
+  ViewMixin, {
+    isComponent: true,
```

Comment:
This is moved from https://github.com/emberjs/ember.js/blob/551a2312038dd815a2ff3e7843a8475da3d0fd11/packages/ember-views/lib/components/component.js#L121